src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,81 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, ColonyClientV6, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeMotionLimits(
+      _,
+      { colonyAddress, userAddress, motionId, tokenDecimals },
+    ) {
+      try {
+        const colonyClient = (await colonyManager.getClient(
+          ClientType.ColonyClient,
+          colonyAddress,
+        )) as ColonyClientV6;
+        const votingReputationClient = await colonyClient.getExtensionClient(
+          Extension.VotingReputation,
+        );
+        const {
+          skillRep,
+          stakes,
+          domainId,
+        } = await votingReputationClient.getMotion(motionId);
+        const { skillId } = await colonyClient.getDomain(domainId);
+        const { reputationAmount } = await colonyClient.getReputation(
+          skillId,
+          userAddress,
+        );
+        // @NOTE There's no prettier compatible solution to this :(
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        // eslint-disable-next-line max-len
+        const userMinStakeFraction = await votingReputationClient.getUserMinStakeFraction();
+
+        const totalStaked = bigNumberify(stakes[1])
+          .div(bigNumberify(10).pow(tokenDecimals))
+          .toNumber();
+        const totalStakeAmount = skillRep
+          .mul(totalStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 2))
```

Comment:
For the native token, this 10 to the power of 36, why ?

This is super wrong

Reply:
See my reasoning below

Reply:
So you're 90% right :)

Talked to Alex, and this should actually be:
```
        const totalStakeAmount = skillRep
          .mul(totalStakeFraction)
          .div(bigNumberify(10).pow(tokenDecimals + 18))
```

Reply:
Yeah, that actually makes more sense :)